src/states/elder/mod.rs

```diff
@@ -706,20 +706,20 @@ impl Elder {
 
     fn handle_filtered_signed_message(
         &mut self,
-        msg: HopMessageWithBytes,
+        mut msg: HopMessageWithBytes,
     ) -> Result<(), RoutingError> {
         trace!(
             "{} - Handle signed message: {:?}",
             self,
-            msg.routing_message()
+            msg.full_message_crypto_hash()
```

Comment:
a hash of message won't help much with debugging, meanwhile the previous whole message will do as it contains searchable key words of network actions.
so here (and for some other trace places as well), we may still need the whole msg to be logged.
if cann't do that, maybe just remove the log ?

Reply:
If you look at the commit, you'll see that when creating a HopeMessageWithBytes from a SignedRoutingMessage we log it, so that first instance of seeing the hash in your log file should contain the full message.

Do you think it won't be enough?

Reply:
Ah, ok.
Yeah, that shall be enough.

Reply:
If this ends up being a hassle it might be worth to write a small utility to pipe the logs through which resolves the hashes.

Reply:
I think it may actually end up quite useful, generally I see the message I care at the start, and then I want to search for it and often I'm missing bits, a hash may be easier in the end :-)